.travis.yml

```diff
@@ -0,0 +1,32 @@
+# http://travis-ci.org/data-exp-lab/widgyts
+language: python
+dist: xenial
+python:
+    - 2.7
+    - 3.5
+    - 3.6
+    - 3.7
+services:
+  - xvfb
```

Comment:
Do we need `xvfb`?  Is this for firefox?

Reply:
I don't think we need it right now! I'll remove it. I referenced the ipywidgets travis config file and I suspect this is there for firefox, yes.